.tekton/github-pr-pipeline.yaml.part

```diff
@@ -26,9 +26,9 @@ spec:
       runAfter:
       - github-set-check-status-to-pending
       - unittest-default
-      - unittest-cassandra
-      - unittest-couchbase
-      - unittest-gevent-starlette
+      #- unittest-cassandra
+      #- unittest-couchbase
+      #- unittest-gevent-starlette
```

Comment:
Why are these commented out?

Reply:
We're not using cassandra, couchbase and gevent-starlette, so @pvital decided to remove related unittests from tekton pipeline.

Reply:
As we spoke with @pvital we're not going to check unittests for cassandra, couchbase and gevent for now in the tekton pipeline so I've removed them.